src/Containers/PanelCard/PanelCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback } from 'react';
+import { Button } from '@Inputs/Button/Button';
+import { CardProps, Card } from '../Card/Card';
+
+type OperationState = { isFailure: boolean } & { isLoading: boolean };
```

Comment:
Remove this. This is intersection not union. Also you don't need either just delete the operationState entirely....

        if (operationState.isFailure) {
            return isFailureMessage;
        }
        if (operationState.isLoading) {
            return isLoadingMessage;
        }
        return isSuccessMessage;


Replace the getMessage function with 

return isFailureMessage || isSuccessMessage || isLoadingMessage